static/app/components/tooltip.tsx

```diff
@@ -466,4 +389,22 @@ const TooltipArrow = styled('span')`
   }
 `;
 
-export default Tooltip;
+interface AcceptanceTestTooltipProxyProps extends TooltipProps {
+  /**
+   * Stops tooltip from being opened during tooltip visual acceptance.
+   * Should be set to true if tooltip contains unisolated data (eg. dates)
+   */
+  disableForVisualTest?: boolean;
+}
+function AcceptanceTestTooltipProxy({
+  disableForVisualTest,
+  ...props
+}: AcceptanceTestTooltipProxyProps) {
+  if (IS_ACCEPTANCE_TEST) {
+    return disableForVisualTest ? null : <AcceptanceTestTooltip {...props} />;
+  }
+
+  return <DO_NOT_USE_TOOLTIP {...props} />;
+}
+
+export default AcceptanceTestTooltipProxy;
```

Comment:
Can we just name it `Tooltip` to make things easy, otherwise we can no-longer auto-complete `Tooltip` from the typescript language server

Reply:
@evanpurkhiser we can export as alias and it'll achieve the same thing, wdyt? I still think it's nice to know that it's just a proxy component, but maybe that's self explanatory

Reply:
Ah fuck @evanpurkhiser :D the tooltip tests rely on the component name 😂 I will name it tooltip